tslearn/docs/user_guide/matrix-profile.rst

```diff
@@ -3,19 +3,28 @@
 Matrix Profile
 ==============
 
-The Matrix Profile, :math:`MP`, is a new time series that can be calculated based on an input time series :math:`T` and a subsequence length :math:`m`. :math:`MP_i` corresponds to the minimal distance from the query subsequence :math:`T_{i\rightarrow i+m}` to :math:`T` (excluding the query subsequence itself) [1]_.  The Matrix Profile is closely related to the distance calculation used to transform time series into their shapelet-transform space, as illustrated below.
+The Matrix Profile, :math:`MP`, is a new time series that can be calculated based on an input time series :math:`T` and a subsequence length :math:`m`. :math:`MP_i` corresponds to the minimal distance from the query subsequence :math:`T_{i\rightarrow i+m}` to :math:`T` [1]_.  As the distance from the query subsequence to itself will be equal to zero, :math:`T_{i-\frac{m}{4}\rightarrow i+\frac{m}{4}}` is considered as an exclusion zone. In order to construct the Matrix Profile, a distance profile which is similar to the distance calculation used to transform time series into their shapelet-transform space, is calculated for each subsequence, as illustrated below:
 
-.. figure:: ../_static/img/distance_matrix_profile.svg
+.. figure:: ../../_images/sphx_glr_plot_distance_and_matrix_profile_001.svg
     :width: 80%
     :align: center
 
     For each segment, the distances to all subsequences of the time series are calculated and the minimal distance that not corresponds to the original location of the segment (where the distance is zero) is returned.
 
 
+Implementation
+---------------
+
+The Matrix Profile implementation provided in ``tslearn`` uses numpy or wraps around STUMPY [2]_. Three different versions are available: 
+* ``numpy``: a slow implementation that supports multi-dimensional data
```

Comment:
These bullets are ill-formatted

Reply:
Nicely spotted :) I am sometimes a little bit clueless on this RST syntax... I guess a blank newline is needed to indicate a new paragraph?